releasetasks/data/docker-worker-pub.pem

```diff
@@ -0,0 +1,31 @@
+-----BEGIN PGP PUBLIC KEY BLOCK-----
```

Comment:
It seems a bit weird to me to embed this into the repo...it's not something tied to the logic of generating release graphs, merely another piece of data needed to do so. What do you think of having this puppet managed, and passing in a reference to it instead?